src/rbac-forms/rbacUsersTable.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import { Table, Checkbox, Icon } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import './styles.scss';
+
+class RbacUsersTable extends Component {
```

Comment:
This component looks great! We don't have to limit ourselves only to rbac and use it in every table since this table has same signature as any table in MiQ. Allow enabling of icons and selectors plus let consumer pass icons/images in first column.

I'd consider moving this to seperate pack perhaps `src/table` so it can be used seperately from forms.